app/views/user_mutes/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<% content_for :heading do %>
+  <h1><%= t ".my muted users" %></h1>
+<% end %>
+
+<%= render :partial => "settings_menu" %>
+
+<h4><%= t ".you_have_muted_n_users", :count => @muted_users.size %></h4>
+<p>
+  <%= t ".user_mute_explainer" %>
+  <em><%= t ".user_mute_admins_and_moderators" %></em>
+</p>
+
+<% if @muted_users.any? %>
+  <table class="table">
+    <thead>
+      <tr>
+        <th><%= t ".muted_user" %></th>
+        <th class="d-flex justify-content-end"><%= t ".actions" %></th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @muted_users.each do |user| %>
+        <tr>
+          <td>
+            <%= user_thumbnail_tiny user %>
+            <%= link_to user.display_name, user_path(user) %>
+          </td>
+          <td class="text-nowrap text-end">
+            <%= link_to t(".unmute"), unmute_user_path(user), :method => :delete, :class => "btn btn-sm btn-primary" %>
+            <%= link_to t(".send_message"), new_message_path(user), :class => "btn btn-sm btn-secondary" %>
```

Comment:
An option to send a message to a muted user seems a bit strange when by definition any reply will wind up in the muted messages and won't be notified?

Reply:
> An option to send a message to a muted user seems a bit strange when by definition any reply will wind up in the muted messages and won't be notified?

I can imagine situations in which I muted another user just to prevent them from spamming my email inbox but still want to keep the conversation open.